### ISSUE TITLE
`dotenvx status` breaks if you have intentionally decrypted .env.X files not part of .env.vault

### ISSUE THREAD

**Issue by nbercurrantsourcream** [dotenvx/dotenvx#194]
Hi first of all great lib 
I ran into the following issue, in our repo we implemented dotenvx and have couple `.env.X` files that are now fully part of `env.vault`

However there are 2 .env files we intend to keep unencrypted as they do not hold anything secret by design.
This does not play nice with `dotenvx status` 

Example:
```
|
|- .env
|- .env.silly_raw
|- .env.vault 
|- .env.keys
```

So .env.vault here has 1 record of the encrypted `.env` as designed and does _not_ have a row for `.env.silly_raw`

```shell
$ dotenvx status
> Cannot read properties of undefined (reading 'split')
``` 
While debugging I narrowed it down to https://github.com/dotenvx/dotenvx/blob/fc283ef2035830366767ed252d940361777c6cf7/src/lib/services/status.js#L66
because for the old data passed down to diff it has undefined. 

I can put up a PR with following local fix;
- add option to `dotenvx status` command `--ignoredFiles <fileName...>'`
- add check here to skip if file name is in the ignoreList https://github.com/dotenvx/dotenvx/blob/fc283ef2035830366767ed252d940361777c6cf7/src/lib/services/status.js#L47

**Comment by motdotla** [comment]
thank you for catching this. looking into your PR, but I might modify the solution to be more convenient for users - so that it only compares to what is in the .env.vault file (and warns on the others). i'll keep you posted.

**Comment by motdotla** [comment]
fixed in `v0.37.1` - releasing shortly.
